ci-operator/config/openshift/frr/openshift-frr-release-main__periodics.yaml

```diff
@@ -0,0 +1,66 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  metallb:
+    name: "4.19"
+    namespace: ocp
+    tag: metallb
+  metallb-frr:
+    name: "4.19"
+    namespace: ocp
+    tag: metallb-frr
+  metallb-operator:
+    name: "4.19"
+    namespace: ocp
+    tag: metallb-operator
+  ocp_builder_rhel-9-base-openshift-4.18:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-base-openshift-4.18
+  ocp_builder_rhel-9-golang-1.23-openshift-4.19:
```

Comment:
why do we need the go image, here we should not build anything, no?

Reply:
let's clean up later together with main